src/Kernel.package/AsciiCharset.class/class/isModifierSymbol..st

```diff
@@ -0,0 +1,3 @@
+character classification
```

Comment:
This has circonflexe and grave, but misses aigu, and arguably cédille and tréma. The latter two are triggered by "," and ":" on the Windows International (soft) keyboard.

Reply:
But those chracters are not in ASCII.  This method is specific to the AsciiCharset class.